api-test/src/main/resources/ida/AddIdentity/AddIdentity.yml

```
@@ -2192,4 +2192,304 @@ AddIdentity:
 }'
       output: '{
   "status":"ACTIVATED"
+}'
+   auth_AddIdentity_For_Kyc_Delegated_Bio_Auth_VID_smoke_Pos:
```

YAMLlint (1.38.0)

[error] duplication of key "auth_AddIdentity_For_Kyc_Delegated_Bio_Auth_VID_smoke_Pos" in mapping

(key-duplicates)